apps/cryptowallet/cryptowallet.yml

```diff
@@ -0,0 +1,11 @@
+name: cryptoWallet
+description: 'electron/node/react based bitcoin wallet.'
```

Comment:
As per the [submission guidelines](https://github.com/electron/electron/issues/12337#issuecomment-432377395), Electron is implied and so it's not necessary to mention Electron in the description (or in the keywords).